server/bin/create-tables.js

```diff
@@ -8,6 +8,8 @@ if (process.argv[2] === 'manual') {
 
 const Client = require('pg').Client;
 
+
+console.log(process.env.DATABASE_URL)
```

Comment:
Prefer to remove this line if possible b/c it leaks secrets (the password).  Was this a debug step that we can take out?